### DISCUSSION TITLE
Kurs: GitHub for Designere

### DISCUSSION THREAD

**Discussion by onordbo** [fremtind/jokul#2505] [Annet]
## Introduksjon 📣 
Etter litt prosessrevolusjon er vi nå godt i gang med nye muligheter for samarbeid på tvers av fagdisipliner innad i Jøkul. Siden vi har flyttet og samlet brorparten av prosessen inn på GitHub og tatt i bruk enda flere funksjoner, det blitt raskt etterspurt å lage et mikrokurs i GitHub for Designere/ikke-tekniske bidragsytere og brukere, slik at vi får å senket terskelen for bidrag, og gjort GitHub som plattform bittelitt mindre skummelt for alle som ikke skriver kode i det daglige.

## Ta kurset på Figma her 👇 
Nå har jeg snekret sammen første versjon av kurset, og det kan tas her—for alle som har en Figma bruker i Fremtind: https://www.figma.com/proto/HOmcpKHfjHnS9f8397hfxU/Github-for-Designere?page-id=0%3A1&node-id=1%3A2&viewport=260%2C48%2C0.1&scaling=scale-down-width&starting-point-node-id=1%3A2

## Kom med masse tilbakemelding, plis 🙏 
Siden dette er første versjon ønsker jeg meg masse tilbakemeldinger på alt dere ser av ting som kan forbedres, eller som dere eventuelt ikke skjønner, eller om det er noe dere føler dere mangler i kurset. Höjta til så fikser jeg!

**Comment by Stormoen** [comment]
Synes dette er superbra! Liker skjermbilder med piler og forklaringer!

**Comment by kristineSorensen** [comment]
Veldig bra kurs @onordbo 
enkelt å følge
